libs/protocol-client/src/lib/utils/ipfs.ts

```diff
@@ -4,17 +4,12 @@ import { ContributionV1SchemaCid } from './constants';
 
 export type MintedContributionSchemaV1 = {
   version: number;
-  schema: `ipfs://QmcmzPR94Vq6U6KxmhK4bNwjekLkZpxeEm2jiMLoYBQ3Xg`;
+  schema: `ipfs://QmSFG2e9RmG9sUTzLzMuamodn4DcNu51RdDzY8cyon8oEx`;
   name: string;
   details: string;
   proof: string;
   activityName: string;
   image?: string; // keep blank for now
-  govrn: {
-    /* Contribution ID */
-    id: number;
-    activityTypeId: number;
-  };
```

Comment:
May I ask why this is removed?

Reply:
Yes, unfortunately when testing I realized that id doesn't work when a contribution is being synced across multiple environments and potentially people could target other peoples contributions to be overwritten.